packages/bun-uws/src/HttpContext.h

```diff
@@ -64,6 +64,33 @@ struct HttpContext {
 
     /* Init the HttpContext by registering libusockets event handlers */
     HttpContext<SSL> *init() {
+        
+        if(SSL) {
```

Comment:
why wasn't this necessary before?

Reply:
Basically BoringSSL will try to auto handshake the TLS connect, but only if enough data was ended, normally HTTP requests have enough data, but the handshake rejection was not working properly before for uWS, this will add same behavior that we expect with fetch and tls minus the checkServerIdentity part, we can add it too if we customize more, but I would suggest doing this in another PR.